src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1022,6 +1022,15 @@ private function registerPropertyInfoConfiguration(array $config, ContainerBuild
 
     private function registerCacheConfiguration(array $config, ContainerBuilder $container, XmlFileLoader $loader)
     {
+        foreach ($config['adapters'] as $name => $adapterConfig) {
+            $adapterDefinition = !empty($adapterConfig['parent']) ? new DefinitionDecorator($adapterConfig['parent']) : $container->getDefinition('cache.adapter.'.$name);
```

Comment:
Keep only the new DecoDef case, since the getDef one is for special cases that we should remove